### PR TITLE
Feature/cpp code

### PR DESCRIPTION
**Pull request by hroest**
Add code for running core algorithm in C++ using RCpp

### CODE REVIEW

File: R/affine_aligned_obj.R [hunk 1]
```diff
@@ -12,6 +12,44 @@
 #' @export
 NeedleObjAffine <- setClass(Class="NeedleObjAffine", representation(Traceback = "list", M = "matrix", A = "matrix", B = "matrix", GapOpen = "numeric", GapExten = "numeric", FreeEndGaps = "logical") )
 
+#' An S4 object for class Person
+#'
+#' @importFrom methods setClass
+#' @export
+Person <- setClass(Class="Person",
```
Comment (hroest) [inline R/affine_aligned_obj.R:19]: do you need this?

File: src/RcppExports.cpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// Generated by using Rcpp::compileAttributes() -> do not edit by hand
```
Comment (hroest) [inline src/RcppExports.cpp:1]: should not be in the repo - it is auto-generated.

**Comment by hroest** [comment]
doesnt go in here:  BIN +779 KB (190%) src/RcppExports.o

File: R/affine_aligned_obj.R (hunk 1 above)
Comment (shubham1637) [reply R/affine_aligned_obj.R:19]: Have been removed

**Pull request closed without merging**